packages/ChangesReloaded-Tests.package/CRFakeFileStream.class/instance/peek.st

```diff
@@ -0,0 +1,6 @@
+accessing
+peek
+
+	self position < self string size 
```

Comment:
```suggestion
	self atEnd not
```

Reply:
@SilvanVerhoeven that must look pretty random, I know :D but I knew that we had that method because @strawberriesandcheese and I implemented it while we were working on the testing stuff together before that whole testing branch got out of hand. And I actually thought we fixed not using our own methods when we noticed those kind of problems within the legacy code. But as it seems, we didn't 🤷‍♀️